middleware/authentication.js

```diff
@@ -0,0 +1,20 @@
+const jwt = require('jsonwebtoken');
+const { UnauthenticatedError } = require('../errors');
+
+const auth = async (req, res, next) => {
```

Comment:
Not a big deal but we don't need the `async` here since we are not awaiting on any asynchronous calls in this function.